modules/setting/setting.go

```diff
@@ -823,6 +842,20 @@ func NewContext() {
 	CSRFCookieHTTPOnly = sec.Key("CSRF_COOKIE_HTTP_ONLY").MustBool(true)
 	PasswordCheckPwn = sec.Key("PASSWORD_CHECK_PWN").MustBool(false)
 
+	sec = Cfg.Section("security.hash")
```

Comment:
`password_hash` is a better section name.

Reply:
without the `security`-prefix?